### PR TITLE
docs: update dead link

### PR DESCRIPTION
**Pull request by alejandrovelez7**
## Context

Was perusing some of the links on the README and this one 404s.

Happy to replace it with an updated link if that exists 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -419,7 +419,6 @@ Articles to help you to get started:
 * [How to end fruitless dev discussions about your project’s code style?](https://medium.com/appaloosa-store-engineering/how-to-end-fruitless-dev-discussions-about-your-project-s-code-style-245070bff6d4)
 * [Free automated code reviews using Pronto](https://hovancik.net/blog/2016/04/11/free-automated-code-reviews-using-pronto/)
 * [Automated Elixir code review with Github, Credo and Travis CI](https://medium.com/fazibear/automated-elixir-code-review-with-github-credo-and-travis-ci-986cd56b8f02)
-* [Running Rubocop before git commit](https://christoph.luppri.ch/articles/2016/11/21/running-rubocop-before-git-commit/)
```
Comment (ashkulz) [inline README.md:422]: What do you think about linking to
https://web.archive.org/web/20181225040512/https://christoph.luppri.ch/articles/code-quality/running-rubocop-before-git-commit/

File: README.md [hunk 2]
```diff
@@ -419,7 +419,6 @@ Articles to help you to get started:
...
 * [Pronto, Codeship and GitHub for automatic code review](http://abinoam.tl1n.com/pronto-codeship-and-github-for-automatic-code-review/)
 * [How to automatically review your PRs for style violations with Pronto and RuboCop](https://christoph.luppri.ch/articles/2017/03/05/how-to-automatically-review-your-prs-for-style-violations-with-pronto-and-rubocop/)
```
Comment (ashkulz) [inline README.md:423]: This link has a redirect in place, but I'd rather we change it to
https://christoph.luppri.ch/how-to-automatically-review-your-prs-for-style-violations-with-pronto-and-rubocop

Comment (alejandrovelez7) [reply README.md:424]: updated!

Comment (ashkulz) [reply README.md:424]: I think you made a mistake, updated the dead link with another one.

Comment (alejandrovelez7) [reply README.md]: ah so sorry I misunderstood what you were suggesting 😄

File: README.md (hunk 1 above)
Comment (alejandrovelez7) [reply README.md:422]: It's a little slow to resolve, but I updated it to this

**Review by ashkulz — APPROVED** [review]

**Comment by ashkulz** [comment]
Thanks for the contribution, @alejandrovelez7!

**Pull request merged**